modules/core/src/test/scala/doobie/util/PutSuite.scala

```diff
@@ -9,6 +9,8 @@ import doobie._
 import scala.concurrent.ExecutionContext
 
 class PutSuite extends munit.FunSuite with PutSuitePlatform {
+  case class X(x: Int)
+  case class Q(x: String)
```

Comment:
Can we add tests for ones with private constructors as well :)

Reply:
added a few more tests now